lib/util/errors.js

```diff
@@ -0,0 +1,55 @@
+exports.makeError = (res, statusCode, err, message) => {
+  // 4 cases:
+  // 1) 'err' is a string
+  // 2) 'err' is a ValidationError
+  // 3) 'err' is a SequelizeValidationError
+  // 4) 'err' is Error
+
+  // If the error is a string, just forward it to user.
+  if (typeof err === 'string') {
+      return res.status(statusCode).json({
+          success: false,
+          message: err
+      });
+  }
+
+  const msgText = message ? message + ' ' + err.message : err.message;
+
+  // If the error is ValidationError, pass the errors details to the user.
+  if (err.name && err.name === 'ValidationError') {
+      return res.status(statusCode).json({
+          success: false,
+          message: msgText,
+          errors: err.errors
+      });
+  }
+
+  // If the error is SequelizeValidationError, pass the errors details to the user.
+  if (err.name && err.name === 'SequelizeValidationError') {
```

Comment:
there's no Sequelize here, therefore this is not needed, right?

Reply:
True